testsuite/N808.py

```diff
@@ -42,8 +42,8 @@
 #: Okay
 Good = TypeVar("Good")
 
-#: N808
-__NotGood = TypeVar("__NotGood")
+#: Okay
+__Good = TypeVar("__Good")
```

Comment:
I disagree. I believe `_T_co` is fine, or `_T`, but I don't believe `__T` (double underscore) is a good example here.

Reply:
```suggestion
_Good = TypeVar("_Good")

#: N808
__NotGood = TypeVar("__NotGood")
```